### ISSUE TITLE
does not replace '.ts' extension on `declare module` statements

### ISSUE THREAD

**Issue by mfulton26** [denoland/dnt#212]
<details>
<summary>Example.ts</summary>

```ts
export default class Example {
  get a() {
    return 42;
  }
}

```

</details>

<details>
<summary>Example.augmentation.ts</summary>

```ts
import Example from "./Example.ts";

declare module "./Example.ts" {
  export default interface Example {
    b: number;
  }
}

Reflect.defineProperty(Example.prototype, "b", {
  value(this: Example) {
    return this.a * 2;
  },
});

```

</details>

```console
% ./path/to/build.ts
[dnt] Transforming...
[dnt] Running npm install...

added 11 packages, and audited 12 packages in 423ms

2 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
[dnt] Building project...
[dnt] Type checking...
src/Example.augmentation.ts:3:16 - error TS2691: An import path cannot end with a '.ts' extension. Consider importing './Example.js' instead.

3 declare module "./Example.ts" {
                 ~~~~~~~~~~~~~~

error: Uncaught (in promise) Error: Had 1 diagnostics.
        throw new Error(`Had ${diagnostics.length} diagnostics.`);
              ^
    at build (https://deno.land/x/dnt@0.30.0/mod.ts:309:15)
    at async file:///path/to/build.ts:13:1
```

I believe that `declare module "./Example.ts" {` should be `declare module "./Example.js" {` in the emitted `src`.

<details>
<summary>emitted src/Example.ts</summary>

```ts
export default class Example {
  get a() {
    return 42;
  }
}

```

</details>

<details>
<summary>emitted src/Example.augmentation.ts</summary>

```ts
import Example from "./Example.js";

declare module "./Example.ts" {
  export default interface Example {
    b: number;
  }
}

Reflect.defineProperty(Example.prototype, "b", {
  value(this: Example) {
    return this.a * 2;
  },
});

```

</details>

**Comment by mfulton26** [comment]
Thank you @dsherret for knocking out so many issues to strengthen `dnt`'s offering; I think this is great!

**Comment by dsherret** [comment]
No problem and sorry for the delay, but I've just been really busy working on the CLI. Hopefully the fix for this one works as I didn't test it in integration (only unit test), but it's released now in 0.31.0
